storage/swift.go

```diff
@@ -116,7 +119,7 @@ func (s *Swift) Info(ctx context.Context, path string) (types.Info, error) {
 	info, _, err := s.conn.Object(ctx, s.cfg.Container, path)
 	if err != nil {
 		if err.Error() == swift.ObjectNotFound.Error() {
-			return types.Info{}, &ObjectNotFound{}
+			return types.Info{}, ObjectNotFound{}
```

Comment:
The `Info` command in s3.go returns `ObjectNotFound{}` as a value (and not a pointer). Let's uniformize as it's creating problems in the calling codes